afqbrowser/site/client/js/tract-details.js

```diff
@@ -293,15 +320,23 @@ function ready(error, data) {
             }
         });
 
+
+
     // compute mean line
     afqb.plots.tractMean = d3.nest()
         .key(function (d) { return d.tractID; })
         .key(function (d) { return d.nodeID; })
         .rollup(function (v) {
-			return d3.mean(v, function (d) {
-				return +d[afqb.controls.plotsControlBox.plotKey];
-			});
-		})
+			 		return{
+						mean: d3.mean(v, function (d) {
+						 return +d[afqb.controls.plotsControlBox.plotKey];}),
+					  stderr: d3.deviation(v, function (d) {
+						 return +d[afqb.controls.plotsControlBox.plotKey];
+					   })/Math.sqrt(afqb.plots.tractData[0].values.length),

```

Comment:
@anotherjoshsmith @arokem Does the length of this have just the subjects in this group or is it a binary grouping variable where the length is as long as all the subjects? Based on appearance of the plots it looks like n is the full n not the group's n